package.json

```diff
@@ -63,5 +63,8 @@
     "test:eslint": "yarn run:concurrent test:eslint",
     "test:unit": "yarn run:sequential test:unit",
     "values": "bin/mdsctl templateValues"
+  },
+  "dependencies": {
+    "chai": "4.2.0"
```

Comment:
This should be up in `devDependencies`

Reply:
Yeah not sure why the deps even changed...

Reply:
It's `mdsctl` -- it attempts to install `chai`. Not sure what line it is on in this PR, but on `develop` you can check out `bin/mdsctl::372`